core/tests/protractor_desktop/preferences.js

```diff
@@ -48,6 +48,22 @@ describe('Preferences', function() {
     expect(preferencesPage.isFeedbackEmailsCheckboxSelected()).toBe(false);
   });
 
+  it('should set and edit bio in user profile', function() {
+    users.createUser('lisa@preferences.com', 'lisaPreferences');
+    users.login('lisa@preferences.com');
```

Comment:
Just a doubt here -- can we use `users.createAndLoginUser(..)` here instead of `users.createUser(..)` and `users.login(..)`?

Reply:
I see the pattern all over e2e tests. Many of the tests do createUser() followed by login(). I think let's defer this change to a later PR? And for now we can merge this PR?

Reply:
Sounds good, agreed. Thanks!